src/main/scala/nl.knaw.dans.easy.file2bag/FilesXml.scala

```diff
@@ -31,27 +31,41 @@ object FilesXml extends DebugEnhancedLogging {
     // easy-deposit-api does not supply rights at all
     // so here we don't supply a default
     // https://github.com/DANS-KNAW/easy-deposit-api/blob/eef71618e8b776fb274da123f8011510499c741a/src/main/scala/nl.knaw.dans.easy.deposit/docs/FilesXml.scala#L40-L42
-    val itemContent = if (accessRights.isBlank) Seq[Node]()
-                      else <accessibleToRights>{ accessRights }</accessibleToRights>
-    val newItem =
-        <file filepath={ "data/" + destinationPath }>
-          <dcterms:format>{ mimeType }</dcterms:format>
-          { itemContent }
-        </file>
-    logger.info(newItem.toOneLiner)
+    var formatTagPrefix = "dcterms"
+    val oldFilesXmlWithPossiblyAddedNamespace =
+      if ((oldFilesXml \ "file" \ "format").nonEmpty) {
+        formatTagPrefix = (oldFilesXml \ "file" \ "format").head.prefix
+        oldFilesXml
+      }
+      else {
+        if (oldFilesXml.scope.getURI(formatTagPrefix) == null) {
```

Comment:
would it not make more sense to always retrieve the namespace here, instead of from an existing `format` element?

Reply:
now done so indeed